package.json

```diff
@@ -76,7 +72,7 @@
     "jest": "^24.8.0",
     "jest-enzyme": "^7.0.2",
     "lerna": "^3.16.5",
-    "marked": "^0.8.0",
+    "marked": "0.7.0",
```

Comment:
I pinned this to the previous version since the new version has engines changed to `">= 8.16.2"`.  There doesn't seem to be a good reason this deep specify this value and I didn't want to change my version of node just to make this dep happy.